libs/utils/utils.js

```diff
@@ -871,8 +871,11 @@ export async function loadArea(area = document) {
     window.dispatchEvent(new Event('milo:LCP:loaded'));
 
     // Post LCP operations.
-    if (isDoc && section.el.dataset.idx === '0') { loadPostLCP(config); }
-
+    if (window.PostLCPevnt && isDoc && section.el.dataset.idx === '0') {
```

Comment:
How about something like `delayPostLCP`, `PostLCPevnt` sounds like an event triggered after LCP, so it might be a bit confusing.

Also, is there a connected consumer PR that actually declares this property?